plugins/chat/assets/javascripts/discourse/components/thread-unread-indicator/index.gjs

```diff
@@ -1,21 +1,32 @@
 import Component from "@glimmer/component";
 
+const MAX_UNREAD_COUNT = 99;
+
 export default class ChatThreadUnreadIndicator extends Component {
   get unreadCount() {
     return this.args.thread.tracking.unreadCount;
   }
 
+  get urgentCount() {
+    return this.args.thread.tracking.watchedThreadsUnreadCount;
+  }
+
   get showUnreadIndicator() {
-    return this.unreadCount > 0;
+    return this.unreadCount > 0 || this.urgentCount > 0;
   }
 
   get unreadCountLabel() {
-    return this.unreadCount > 99 ? "99+" : this.unreadCount;
+    const count = this.urgentCount > 0 ? this.urgentCount : this.unreadCount;
+    return count > MAX_UNREAD_COUNT ? `${MAX_UNREAD_COUNT}+` : count;
```

Comment:
Unrelated to this PR but I wonder if that notation (`99+`) works in all locale? 🤔

Reply:
That's a good question, we have this in several places for notifications too. This is used as a badge on top of icons within chat so I think the risk would also be that using words to describe it (ie. "more than 99") wouldn't be an option when translated as it will break the UI. Though maybe showing the actual count would work.